### PR TITLE
Fix #1723 - Show missing fields in Edit Saving Application

### LINKED ISSUE #1723: On editing existing weekly charge, it doesn't show the repeats every field hence cant edit repeats every.
Reported by Subramanya at https://mifosforge.jira.com/browse/MIFOSX-2557
Original Description:
>> Create savings account with weekly charge, 
>> Click on submit, to submit new savings application, 
>> click on Undo approval, 
>> In the modification savings page, for weekly charge - Repeats every field is not available. 
Expected:- 
>> Repeats every field should be displayed for weekly charge on the modification page.

### PR DESCRIPTION
**Pull request by AkashBhave**
## Description
Added the ability to view and edit two fields: **Date** and **Repayments Every**. These fields were not displayed before, which meant that a user could not edit them either. As expected, the drop-down menu shows when entering a date.

## Screenshots
ss

<hr>
Part of a task for Google Code-in 2017. 🥇

### CODE REVIEW

**Comment by mbj36** [comment]
Thanks for the PR, It will be reviewed Shortly!

(Powered by [GitMate.io](https://gitmate.io))

**Comment by gkrishnan724** [comment]
@AkashBhave could you please claim the GCI task for this

**Comment by AkashBhave** [comment]
@gkrishnan724 I claimed it about 2 hours ago.

**Comment by AkashBhave** [comment]
@gkrishnan724 Ah, I understand what the issue is. Thanks for the tip! 😄 

I have updated my pull request with the changes you requested, and everything should be working now (including the request payload).

File: app/views/savings/edit_saving_account_application.html [hunk 1]
```diff
@@ -262,27 +262,26 @@ <h4>{{ 'label.heading.editsavingapplication' | translate }}</h4>
...
-                               ng-model="charge.dueDate" is-open="opened"/>
-                    </td>
-                    <td ng-show="charge.chargeTimeType.value=='Monthly Fee'">
+                               ng-model="charge.dueDate" is-open="opened" ng-switch-when='Specified due date'/>
                         <input readonly class="date-disable" class="input-medium form-control" readonly
                                class="date-disable" type="text" datepicker-pop="dd MMMM"
-                               ng-model="charge.feeOnMonthDay" is-open="opened"/>
+                               ng-model="charge.feeOnMonthDay" is-open="opened" ng-switch-when='Monthly Fee'/>
+                        <input id="charges[{{$index}}].dueDate" class="input-medium form-control" readonly
+                               class="date-disable" type="text" datepicker-pop="dd MMMM yyyy"
```
Comment (gkrishnan724) [inline app/views/savings/edit_saving_account_application.html]: why is this `date-disable`?

**Comment by AkashBhave** [comment]
Removed `class="date-disable"` and `readonly` to the field. The field is now editable and can be submitted.

**Pull request merged**